packages/opencensus-exporter-zpages/src/zpages-frontend/page-handlers/tracez.page-handler.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2018 OpenCensus Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as tracing from '@opencensus/nodejs';
+import {types} from '@opencensus/opencensus-core';
+import * as fs from 'fs';
+
+import {TraceMap, ZpagesExporter} from '../../zpages';
+import {LatencyBucketBoundaries} from '../latency-bucket-boundaries';
+import {PageHandler} from '../types';
+
+const ejs = require('ejs');
+const timeunit = require('timeunit');
+
+export class TracezPageHandler implements PageHandler {
+  /** HTML page */
+  private html: string;
+
+  constructor() {
+    this.html = '';
+  }
+
+  /**
+   * Generate Zpages Tracez HTML Page
+   * @param params values to put in HTML template
+   * @returns Output HTML
+   */
+  emitHtml(params?): string {
+    const tracezFile =
```

Comment:
Yes, it does. But ejs use the ```fs.readFileSync``` as default, so in background it runs the same method. Anyway I'll replace it, because we'll remove a not necessary import.